AnnotatoBackend/Sources/App/SharedLibraryModels+Persistable/Document+Persistable.swift

```diff
@@ -0,0 +1,10 @@
+import AnnotatoSharedLibrary
+
+extension Document: Persistable {
+    typealias PersistedEntity = DocumentEntity
```

Comment:
I think this is not used anywhere.

Reply:
This is just to explicitly state the associated type, but you're right that it's not used since Swift can infer this based on the method signatures